r/tools/linuxlibs.R

```diff
@@ -315,8 +315,13 @@ build_libarrow <- function(src_dir, dst_dir) {
 }
 
 ensure_cmake <- function() {
-  cmake <- Sys.which("cmake")
-  if (!nzchar(cmake) || cmake_version() < 3.2) {
+  cmake <- find_cmake(c(
+    Sys.getenv("CMAKE"),
+    Sys.which("cmake"),
+    Sys.which("cmake3")
```

Comment:
Does `cmake3` actually exist in the wild?

Reply:
Apparently so; it was in the report on https://issues.apache.org/jira/browse/ARROW-9602